kuma/wiki/templates/wiki/document.html

```diff
@@ -58,8 +59,8 @@
   <link rel="alternate" type="application/json" href="{{ url('wiki.json_slug', document.slug) }}">
   <link rel="canonical" href="{{ canonical }}" >
 
-  {% if is_translated %}
-    {% for translation in document.other_translations %}
+  {% if other_translations %}
```

Comment:
This `if` will not work correctly since `document.other_translations ` is the related manager and not some truthy object. You must use `exists()`